### PR TITLE
Feature: Modify flatbuffer processing to arrow record processing

### PR DESCRIPTION
**Pull request by liikeuforever**
# Required for all PRs

- [ ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #

### CODE REVIEW

File: plugins/inputs/cnosdb/telegraf.conf
Comment (zipper-meng) [inline plugins/inputs/cnosdb/telegraf.conf:1]: This config file is needless

File: plugins/inputs/cnosdb/kv_service.go [hunk 1]
```diff
@@ -32,97 +31,96 @@ func NewTSKVService(acc telegraf.Accumulator) service.TSKVServiceServer {
...
-							v := field.ValueBytes()
-							fields[string(schema.FieldName(fieldI))] = v[0] == byte(1)
-						case models.FieldTypeString:
-							v := string(field.ValueBytes())
-							fields[string(schema.FieldName(fieldI))] = v
-						default:
-							// Do nothing ?
-						}
+				for j := 0; j < int(numCols); j++ {
+					switch columnIndexToType[uint64(j)] {
```
Comment (zipper-meng) [inline plugins/inputs/cnosdb/kv_service.go:114]: Missing variants of time column, such as  `TIME_TIMESTAMP(NANOSECOND)`

File: plugins/inputs/cnosdb/kv_service.go [hunk 2]
```diff
@@ -32,97 +31,96 @@ func NewTSKVService(acc telegraf.Accumulator) service.TSKVServiceServer {
...
+						fields[columnIndexToName[j]] = colArrMap[j].(*array.Int64).Value(i)
+					case "FIELD_BIGINT UNSIGNED":
+						fields[columnIndexToName[j]] = colArrMap[j].(*array.Uint64).Value(i)
+					case "FIELD_DOUBLE":
+						fields[columnIndexToName[j]] = colArrMap[j].(*array.Float64).Value(i)
+					case "FIELD_BOOLEAN":
+						fields[columnIndexToName[j]] = colArrMap[j].(*array.Boolean).Value(i)
 					}
 				}
 				s.accumulator.AddFields(tab, fields, tags)
```
Comment (zipper-meng) [inline plugins/inputs/cnosdb/kv_service.go]: Maybe you can use accumulator.AddMetric to send timestamps extracted from the source:
```go
if hasTimestamp {
  s.accumulator.AddMetric(metric.New(tab, tags, fields, time))
} else {
  s.accumulator.AddFields(tab, fields, tags)
}
```

File: plugins/inputs/cnosdb/telegraf.conf
Comment (liikeuforever) [reply plugins/inputs/cnosdb/telegraf.conf:1]: Done.

File: plugins/inputs/cnosdb/kv_service.go (hunk 1 above)
Comment (liikeuforever) [reply plugins/inputs/cnosdb/kv_service.go:114]: Done.

**Pull request merged**